### ISSUE TITLE
Include virtual not working on local php server.

### ISSUE THREAD

**Issue by HarryAdney** [brapifra/vscode-phpserver#40]
Works fine on the live server but the footer doesn't display on localhost:3000.

`<!--#include virtual="./includes/footer.php" -->`

Using Version 2.4.4

**Comment by brapifra** [comment]
That's ASP code, not PHP.
You should use `<?php include './includes/footer.php';?>`

**Comment by HarryAdney** [comment]
Ahh, what an idiot I am.

Thanks

**Comment by HarryAdney** [comment]
OK, so I tested this;
Using `<!--#include virtual="./includes/footer.php" -->` works on the live server (NOT VScode's Live server extension; the actual web server the site is stored on), but it doesn't' work on the localhost using the vscode php server extension.

HOWEVER

Using `<?php include './includes/footer.php';?>` works on localhost, using the PHP server, but not on the live server.

To get this to work on both I've added the footer twice, like so:
`</section>`
`</main>`
`    <!--#include virtual="./includes/footer.php" -->`
    `<?php include './includes/footer.php'; ?>`
`</body>`

On viewing the browser source code, the footer is only present once in each case, even though the actual HTML has both includes added, as shown above.
